web/src/components/organisms/Project/Content/hooks.ts

```diff
@@ -0,0 +1,15 @@
+import { useGetItemsQuery } from "@reearth-cms/gql/graphql-client-api";
+import { useModel } from "@reearth-cms/state";
+
+export default () => {
```

Comment:
Is this being used? Its a new file but the Content/index isn't using it?

Reply:
@KaWaite  It is shared and used by the two child hooks
`Content/ContentDetails/hooks`
`Content/ContentList/hooks`